sros2/test/sros2/commands/security/verbs/test_generate_policy_no_nodes.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Canonical Ltd
+# Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
File has multiple copyright statements but no history

Reply:
That's a very good point. It is work derived from a source file that was originally developed by Canonical. Do tell how we should proceed with this.

Reply:
Should it actually share git history with another file? Or is it more of a "I copied that file and turned it into something else" situation?

Reply:
It's a modified copy of `test_generate_policy.py`.